frontend/src/pages/index.vue

```diff
@@ -66,31 +66,31 @@ if (userLibraries.isReady) {
 const homeSections = computed<HomeSection[]>(() => {
   // Filter for valid sections in Jellyfin Vue
   // TODO: Implement custom section order
-  let homeSectionsArray = pickBy(
+  const homeSectionPrefs = pickBy(
     // @ts-expect-error - No typings for this
-    clientSettings.CustomPrefs,
-    (value: string, key: string) => {
+    clientSettings.CustomPrefs as { [key: string]: string },
```

Comment:
This does add an `as` cast, but I think it makes sense here, since we do not have typings for `CustomPrefs` and the cast is used to indicate to the reader what the value is expected to be.

Reply:
No worry too much about this, this whole section needs refactoring anyway (it doesn't work at all), so it's good!